stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/disposable/DisposableExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.getstream.sdk.chat.disposable
+
+/**
+ * Helper function to add a new disposable to disposableList
+ *
+ * @param disposableList [DisposableList]
+ */
+public fun Disposable.addToDisposableList(disposableList: DisposableList) {
```

Comment:
Do we really need this extension?
And do we need to expose it to our customers?

It is a bit weird that a variable that is received as a parameter is the one that is updated.

I have seen it is used to have some sugar API when a method return a disposable that needs to be added.
```
val disposableList = DisposableList()
createDisposable().addToDisposableList(disposableList)
```

It can be switch by:
```
val disposableList = DisposableList()
disposableList.add(createDisposable())
```
Or if you would need to concatenate new actions, you could use:
```
val disposableList = DisposableList()
createDisposable().also(disposableList::add)
```